extensions/goose/src/test-snapshots/gemini.yaml

```
@@ -63,12 +63,12 @@ data:
-    
+
     # Required for headless mode
     prompt: "Plan a 7 days trip in Paris, mention per day the planning, starting on monday."
-    
+
     instructions: |
       Your task is to:
       1. List the important monuments
       2. Plan each day of the trip
-          
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)